go.mod

```diff
@@ -1,14 +1,15 @@
 module github.com/flagship-io/flagship-common
 
-go 1.21
+go 1.22.0
```

Comment:
hmm.. should probably be  1.23.1 or 1.23.2

just removing the toolchain if it's superior to the go directive version might break the compilation on some system ^^ you should harmonize it since it seems a dependency was requiring 1.23.1 :)